src/components/main/actionsPanel/navigatorPanel/components/DefaultPanel.tsx

```diff
@@ -0,0 +1,95 @@
+import React, { useContext, useMemo } from "react";
+import { useSelector } from "react-redux";
+
+import { SVGIconI } from "@_components/common";
+import { TFileNodeData } from "@_node/file";
+import { MainContext, navigatorSelector } from "@_redux/main";
+
+import { useNavigatorPanelHandlers } from "../hooks";
+import { isHomeIcon, getFileNameFromPath } from "../helpers";
+
+export const DefaultPanel = React.memo(() => {
+  const { file } = useSelector(navigatorSelector);
+  const { project, ffTree, filesReferenceData } = useContext(MainContext);
+
+  const node = useMemo(() => ffTree[file.uid], [ffTree, file.uid]);
+  const fileName = useMemo(() => getFileNameFromPath(file), [file]);
+
+  const { onProjectClick, onFileClick } = useNavigatorPanelHandlers();
+
+  return (
+    <>
+      <div className="gap-s align-center" onClick={onProjectClick}>
+        <SVGIconI {...{ class: "icon-xs" }}>folder</SVGIconI>
+
+        <span
+          className="text-s"
+          style={{
+            maxWidth: "60px",
+            textOverflow: "ellipsis",
+            whiteSpace: "nowrap",
+            overflow: "hidden",
+          }}
+        >
+          {project.name}
+        </span>
+      </div>
+
+      <span className="text-s opacity-m">/</span>
+
+      {/* path */}
+      {file.parentUid !== "ROOT" && (
+        <>
+          <span className="text-s">...</span>
+          <span className="text-s opacity-m">/</span>
+        </>
+      )}
+
+      {/* file */}
+      {node && (
+        <div className="gap-s align-center" onClick={onFileClick}>
+          <SVGIconI {...{ class: "icon-xs" }}>
+            {isHomeIcon(node)
+              ? "home"
+              : filesReferenceData[
+                  (node.data as TFileNodeData).ext.substring(
+                    1,
+                    (node.data as TFileNodeData).ext.length,
+                  )
+                ] &&
+                (node.data as TFileNodeData).ext.substring(
+                  1,
+                  (node.data as TFileNodeData).ext.length,
```

Comment:
(node.data as TFileNodeData).ext.substring(
                  1,
                  (node.data as TFileNodeData).ext.length,
                )
                
                break this into a function